spec/refile/cloudinary_spec.rb

```diff
@@ -1,7 +1,27 @@
 require "spec_helper"
+require "refile/cloudinary"
+
+RSpec.describe Refile::Cloudinary do
+  let(:config) { {
+    cloud_name: "hfkkd49r3",
+    api_key: "some-key",
+    api_secret: "some-secret"
+  } }
 
-describe Refile::Cloudinary do
   it "has a version number" do
     expect(Refile::Cloudinary::VERSION).not_to be nil
   end
+  describe "behavior Refile::Cloudinary" do
+    let(:backend_default) { Refile::Cloudinary.new(**config) }
```

Comment:
Please remove the `**`, they are not necessary and might be confusing to people looking at those.